snuba/datasets/configuration/issues/storages/search_issues.yaml

```diff
@@ -94,7 +94,6 @@ delete_allocation_policies:
     args:
       required_tenant_types:
         - referrer
-        - project_id
```

Comment:
This is intentional - now that we have the batching approach, the allocation policy won't have a `project_id` because it will be (potentially) multiple `project_id`s, so we can ratelimit based on that. This allocation policy will work similar to the subscriptions executor where there is one referrer for those queries. Though the referrer below will be based on storage.